CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased](https://github.com/Instabug/Instabug-Flutter/compare/v12.7.0...dev)
+
+### Added
+
+- Add ``` SessionReplay.getSessionReplayLink ``` API which retrieves the current session's replay link ([#445](hhttps://github.com/Instabug/Instabug-Flutter/pull/445)).
```

Comment:
I think we can change the SessionReplay.getSessionReplayLink code block to code block to inline code.
```suggestion
- Add `SessionReplay.getSessionReplayLink` API which retrieves the current session's replay link ([#445](hhttps://github.com/Instabug/Instabug-Flutter/pull/445)).
```